internal/pkg/api/middleware/auth_test.go

```
@@ -290,68 +290,68 @@ func TestJWTLogoutIntegration(t *testing.T) {
-		
+
 		if claims.UserID != userID {
 			t.Errorf("Expected user ID %d, got %d", userID, claims.UserID)
 		}
 		if claims.SessionID != sessionID {
 			t.Errorf("Expected session ID %d, got %d", sessionID, claims.SessionID)
 		}
-		
+
 		// Simulate logout by "deactivating" the session (this would happen in DB)
 		sessionActive := true
```

golangci-lint (1.64.8)

ineffectual assignment to sessionActive

(ineffassign)